### PR TITLE
Undo tracking of .def files as sources

### PR DESCRIPTION
**Pull request by dnkoutso**
My initial pull request was wrong at https://github.com/CocoaPods/CocoaPods/pull/4490 and they are meant to be treated as header files instead of source files.

Correct xcodeproj PR https://github.com/CocoaPods/Xcodeproj/pull/338

Apologies for this.

### CODE REVIEW

**Comment by segiddins** [comment]
Could you update this to point to Xcodeproj master, and rebase to get rid of the merge commit? Thanks!

**Comment by dnkoutso** [comment]
Sure thing!

**Comment by dnkoutso** [comment]
Done.

File: spec/unit/sandbox/file_accessor_spec.rb [hunk 1]
```diff
@@ -237,7 +237,7 @@ module Pod
           file_patterns = ['Classes/*.{h,m,d}', 'Vendor', 'framework/Source/*.h']
           options = {
             :exclude_patterns => ['Classes/**/osx/**/*', 'Resources/**/osx/**/*'],
-            :dir_pattern => '*{.m,.mm,.i,.c,.cc,.cxx,.cpp,.c++,.swift,.def,.h,.hh,.hpp,.ipp,.tpp,.hxx}',
+            :dir_pattern => '*{.m,.mm,.i,.c,.cc,.cxx,.cpp,.c++,.swift,.h,.hh,.hpp,.ipp,.tpp,.hxx}',
```
Comment (segiddins) [inline spec/unit/sandbox/file_accessor_spec.rb]: shouldn't this now contain `.def` in the headers list?

Comment (dnkoutso) [inline spec/unit/sandbox/file_accessor_spec.rb]: duh yes, updated.

Comment (dnkoutso) [inline spec/unit/sandbox/file_accessor_spec.rb]: Actually because `file_accessor_spec` is already merged like this, it wont show up in the diff.

Comment (segiddins) [inline spec/unit/sandbox/file_accessor_spec.rb]: https://travis-ci.org/CocoaPods/CocoaPods/jobs/90870872#L1673-L1676

**Comment by dnkoutso** [comment]
le green!

**Pull request merged**